### DISCUSSION TITLE
ssl_cert_check as external check

### DISCUSSION THREAD

**Discussion by rmaeder** [selivan/https-ssl-cert-check-zabbix#15] [Ideas]
The script also works very well as a zabbix server external check, so you don't need to set up user parameters, if the monitored port is reachable from the server. Items then look like this:
```
ssl_cert_check.sh[expire,{HOST.CONN},443,www.example.com]
```

**Comment by iRomanyshyn** [comment]
I'm using it right that way. Also, if someone interested, I can share my
template ready to import.

сб, 13 лют. 2021, 11:16 користувач Roman Mäder <notifications@github.com>
пише:

**Comment by rubenjimenezit** [comment]
I was having difficulties to configure this integration with Zabbix until I found that the host had to have Zabbix agent installed or otherwise I should use the Zabbix server (127.0.0.1) as host for the template that is provided

However, configuring the item as an External check like this works like a charm and can be applied to all my already existent hosts whether they user Zabbix agent or not

Thank you @rmaeder !

**Reply by rubenjimenezit** [reply]
Item example. Created inside a fresh new template
Assign the template to a host to see the results

To check validity of the certificate, just substitute "expire" for "valid" in the Key
 
![image](https://github.com/user-attachments/assets/2058193f-e37b-45f0-a2cc-cc2fd365ad25)
